pkg/ccl/backupccl/backupinfo/manifest_handling.go

```diff
@@ -940,6 +948,30 @@ func ValidateEndTimeAndTruncate(
 	)
 }
 
+// ElideSkippedLayers removes backups that are skipped in the backup chain.
+func ElideSkippedLayers(
```

Comment:
I think it'd be good to leave a comment confirming the order of the backup manifests in the slice. I'm assuming based on the function that the manifests are ordered by `end_time` ascending.

Reply:
I don't think this function changes the order, and I think anything we call a "chain" as we do in this comment is generally assumed to be end-time ordered, right?

Reply:
Not something I want to do in this small change, but maybe should introduce `type Chain []Manifest` that can both serve as a place to put docs like this and also to organize various methods/behaviors of chains (like truncation).